packages/manager/src/components/Divider.tsx

```diff
@@ -0,0 +1,33 @@
+import Divider, { DividerProps as _DividerProps } from '@mui/material/Divider';
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import { isPropValid } from 'src/utilities/isPropValid';
+
+export interface DividerProps extends _DividerProps {
+  dark?: boolean;
+  light?: boolean;
+  spacingTop?: number;
+  spacingBottom?: number;
+}
+
+const _Divider = (props: DividerProps) => {
+  return <StyledDivider {...props} />;
+};
+
+export default _Divider;
+
+const StyledDivider = styled(Divider, {
+  label: 'StyledDivider',
+  shouldForwardProp: (prop) =>
+    isPropValid(['spacingTop', 'spacingBottom'], prop),
```

Comment:
Should `light` and `dark` also be filtered out? 

```suggestion
    isPropValid(['spacingTop', 'spacingBottom', 'light', 'dark'], prop),
```

Reply:
I don't mind adding them to the array, but I didn't see a console warning about them.